File: module-contracts/src/main/java/com/beat/contracts/notification/BookingNotification.java
package com.beat.contracts.notification;

import java.time.LocalDateTime;

public record BookingNotification(
	LocalDateTime bookingDateTime,
	String performanceTitle,
	int purchaseTicketCount,
	String bookerName,
	String scheduleDisplayName,
	int currentSoldTicketCount,
	int totalTicketCount
) {
}

File: module-contracts/src/main/kotlin/com/beat/contracts/notification/BookingNotification.kt
package com.beat.contracts.notification

import java.time.LocalDateTime

data class BookingNotification(
    val bookingDateTime: LocalDateTime,
    val performanceTitle: String,
    val purchaseTicketCount: Int,
    val bookerName: String,
    val scheduleDisplayName: String,
    val currentSoldTicketCount: Int,
    val totalTicketCount: Int,
)
